assets/stylesheets/nhsuk-print.scss

```diff
@@ -29,6 +31,14 @@ body {
 }
 
 a {
+  color: $black !important;
+  font-weight: 700;
```

Comment:
I think underline affects legibility, especially for words with plenty of descenders. I made it bold because in my opinion it works better. Besides underlined text is intended to differentiate it from normal text so user can easily see that it's an interactive item. On paper it's not important and the URL is followed in parentheses so legibility is more important.

I can't remember the last time I saw an underlined link in a book.